pandevice/userid.py

```diff
@@ -299,6 +299,19 @@ def get_registered_ip(self, ip=None, tags=None, prefix=None):
             dict: ip addresses as keys with tags as values
 
         """
+        start = 1
+        limit = 500
+        addresses = {}
+        while True:
+            newaddr = self.get_registered_ip_loop(ip, tags, prefix, start, limit)
+            rescnt = len(newaddr.keys())
```

Comment:
"recent" is misspelled

Reply:
rescnt actually is shorthand for resultCount ...